### PR TITLE
Add DB script changes related to API Product

### PR DESCRIPTION
**Pull request by vithu30**

### CODE REVIEW

File: modules/distribution/resources/migration-2.6.0_to_3.0.0/db2.sql [hunk 1]
```diff
@@ -68,3 +68,17 @@ CREATE TABLE IF NOT EXISTS AM_NOTIFICATION_SUBSCRIBER (
...
+
+CREATE TABLE AM_API_PRODUCT_MAPPING (
+  API_PRODUCT_MAPPING_ID INTEGER GENERATED ALWAYS AS IDENTITY (START WITH 1 INCREMENT BY 1),
+  API_ID INTEGER,
+  URL_MAPPING_ID INTEGER,
+  FOREIGN KEY (API_ID) REFERENCES AM_API(API_ID) ON DELETE CASCADE,
+  FOREIGN KEY (URL_MAPPING_ID) REFERENCES AM_API_URL_MAPPING(URL_MAPPING_ID) ON DELETE CASCADE,
+  PRIMARY KEY(API_PRODUCT_MAPPING_ID)
+)
+  /
```
Comment (npamudika) [inline modules/distribution/resources/migration-2.6.0_to_3.0.0/db2.sql]: Shall we fix the minor formatting issue here?

**Comment by npamudika** [comment]
This PR contains fixes related to https://github.com/wso2/product-apim/issues/5464

**Comment by npamudika** [comment]
This PR contains fixes related to https://github.com/wso2/product-apim/issues/5464

File: modules/distribution/resources/migration-2.6.0_to_3.0.0/db2.sql (hunk 1 above)
Comment (vithu30) [reply modules/distribution/resources/migration-2.6.0_to_3.0.0/db2.sql]: Resolved

**Pull request merged**